packages/autorest.go/package.json

```diff
@@ -43,6 +43,11 @@
   },
   "homepage": "https://github.com/Azure/autorest.go#readme",
   "readme": "https://github.com/Azure/autorest.go/blob/main/readme.md",
+  "workspaces": [
```

Comment:
do you actually need this, this feels super weird to me. As this is meant to define this package as a workspace(naming is terrible) and those as project in the workspace

Reply:
Ah I see. Removed.